src/styles/card.scss

```diff
@@ -0,0 +1,97 @@
+.card {
+  margin: 0;
+  padding: 0;
+  font-family: 'Roboto', sans-serif;
+  font-weight: 400;
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin-top: 32px;
+  }
+
+  &__name-panel {
+    width: 166px;
```

Comment:
No need to hardcode width for the content, add padding for the card and use everywhere width 100%